data/tokens.json

```diff
@@ -3021,6 +3021,253 @@
       "type": "spacing"
     }
   },
+  "inputs": {
+    "gtmds_input_size-height-sm": {
+      "value": "{gtmds_size-lg}*2",
+      "type": "sizing"
+    },
+    "gtmds_input_size-height-lg": {
+      "value": "{gtmds_size-xl}+{gtmds_size-lg}",
+      "type": "sizing"
+    },
+    "gtmds_input_size-width": {
+      "value": "auto",
+      "type": "sizing"
+    },
+    "gtmds_input_margin-right": {
+      "value": "{gtmds_space-xxs}",
+      "type": "spacing"
+    },
+    "gtmds_input-content_margin-right": {
+      "value": "{gtmds_space-xxs}/2",
+      "type": "spacing"
+    },
+    "gtmds_input_margin-bottom": {
+      "value": "{gtmds_space-xxs}/2",
+      "type": "spacing"
+    },
+    "gtmds_input_padding-left": {
+      "value": "{gtmds_space-xxs}",
+      "type": "spacing"
+    },
+    "gtmds_input_padding-right": {
+      "value": "{gtmds_space-xxs}",
+      "type": "spacing"
+    },
+    "gtmds_input_padding-vertical-sm": {
+      "value": "{gtmds_space-xxs}/2",
+      "type": "spacing"
+    },
+    "gtmds_input_padding-vertical-lg": {
+      "value": "{gtmds_space-xxs}",
+      "type": "spacing"
+    },
+    "Chip": {
+      "gtmds_input_margin-row-bottom": {
+        "value": "{gtmds_space-xxs}/2",
+        "type": "spacing"
+      },
+      "gtmds_input_margin-row-right": {
+        "value": "{gtmds_space-xxs}/2",
+        "type": "spacing"
+      },
+      "gtmds_input_padding-left-sm": {
+        "value": "{gtmds_space-xxs}/2",
+        "type": "spacing"
+      }
+    },
+    "Invisible-box": {
```

Comment:
i think this can be removed as a token and just be a manual setting

Reply:
Probably this can referee to the previous comment as well. I am not sure which one is the right approach maybe Tsetso can validate this?

Reply:
Yes, let's figure this out together tomorrow :)